OxfordBodleian/Juel-Jensen/BDLaethg37.xml

```diff
@@ -132,6 +447,17 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                         <dim type="right">10</dim>
                                         <dim type="left">9</dim>
                                     </dimensions>
+                                    <note>The characters per line are 13-16.</note>
+                                    <ab type="ruling" subtype="pattern">
+                                        <locus from="6ra" to="53vb"/>    
+                                        Ruling pattern:
+                                        1A-1A-1A1A/0-0/0-0/C.</ab>
+                                    <ab type="pricking">Primary pricks are visible.</ab>
+                                    <ab type="pricking">Ruling pricks are visible.</ab>
+                                    <ab type="ruling">The upper line is written above the
+                                        ruling.</ab>
+                                    <ab type="ruling">The bottom line is written above the
+                                        ruling.</ab>
                                 </layout>
                                 
                                 <layout columns="1" writtenLines="2 18">
```

Comment:
Could ypu provide the range of folia?

Reply:
I provided the folia under `<ab type="ruling" subtype="pattern">` - would you suggest them under another ruling related element or repeated?

Reply:
No, I must missed it above! Sorry!